rllib/algorithms/algorithm.py

```diff
@@ -1651,11 +1673,27 @@ def restore_workers(self, workers: EnvRunnerGroup) -> None:
             )
 
             # Fire the callback for re-created workers.
-            self.callbacks.on_workers_recreated(
-                algorithm=self,
-                worker_set=workers,
-                worker_ids=restored,
-                is_evaluation=workers.local_env_runner.config.in_evaluation,
+            make_callback(
+                "on_env_runners_recreated",
+                callbacks_objects=self.callbacks,
+                callbacks_functions=self.config.callbacks_on_env_runners_recreated,
+                kwargs=dict(
+                    algorithm=self,
+                    env_runner_group=workers,
+                    env_runner_indices=restored,
+                    is_evaluation=workers.local_env_runner.config.in_evaluation,
+                ),
+            )
+            # TODO (sven): Deprecate this call.
+            make_callback(
```

Comment:
Why deprecating it? This might be useful for metrics, wouldn't it?

Reply:
Oh, no, this is just the renamed one, in case users still implement the old name:
`on_workers_recreated` -> `on_env_runners_recreated`